### ISSUE TITLE
Moving appimaged AppImage file makes everything useless

### ISSUE THREAD

**Issue by machitgarha** [probonopd/go-appimage#142]
When I installed appimaged, it was located in my home (i.e. `~/`). Then, for some reason, I moved it to somewhere else, e.g. `~/Downloads`. This, unfortunately, makes (1) desktop files and (2) the daemon unusable, because the desktop files and daemon actions (e.g. `systemctl --user restart appimaged.service`) rely on the absolute path of appimaged determined at installation time.

Although it should not happen often, it confuses the user, as it no longer works correctly.

**Comment by probonopd** [comment]
Indeed. This is yet another point in case that illustrates that the concept of desktop files is insufficient.

A better approach would be for file managers to support AppImages natively, without the need for desktop files.

**Comment by almereyda** [comment]
In some way this can be considered a special case of #162, just that `appimaged` is also not capable of adjusting its own paths in the `appimagekit_*.desktop` files it manages.
